Core/GDCore/Extensions/Metadata/ParameterMetadataTools.cpp

```diff
@@ -108,8 +112,14 @@ void ParameterMetadataTools::ParametersToObjectsContainer(
     }
 
     auto& object = outputObjectsContainer.GetObject(objectName);
+    const auto& allBehaviorNames = allObjectNonDefaultBehaviorNames[objectName];
     for (const auto& behaviorName : object.GetAllBehaviorNames()) {
-      const auto& allBehaviorNames = allObjectBehaviorNames[objectName];
+      if (object.GetBehavior(behaviorName).IsDefaultBehavior()) {
+        // Default behaviors are already ensured to be all present
+        // (and no more than required by the object type).
+        continue;
```

Comment:
Actually, I think it's legit to add capabilities on an object without any specified type.